src/middlewared/middlewared/plugins/init_shutdown_script.py

```diff
@@ -104,33 +103,50 @@ class Config:
         if verrors:
             raise verrors
 
+    @private
+    async def execute_task(self, task):
+        task_type = task['type']
+        cmd = None
+
+        if task_type == 'COMMAND':
+            cmd = task['command']
+        elif os.path.exists(task['script'] or '') and os.access(task['script'], os.X_OK):
+            cmd = f'exec {task["script"]}'
+
+        if cmd:
+            proc = await Popen(
+                cmd,
+                stdout=subprocess.PIPE,
```

Comment:
WIth same result we can set it to `subprocess.DEVNULL` because we are not using the output anyway

Reply:
Or log the stderr ; )